src/main/resources/org/jenkinsci/plugins/mesos/MesosAgentSpecTemplate/ContainerInfo/help-overrideShellCommand.html

```diff
@@ -0,0 +1,7 @@
+<div>
+  Check if you do not want to use the default command to launch the Jenkins agent. This is useful if
+  you want to start the agent in a Windows container:
+  <code>
+    java -DHUDSON_HOME=jenkins -server -Xmx%dm %s -jar ${MESOS_SANDBOX-.}/agent.jar %s %s -jnlpUrl %s";
```

Comment:
Kai, do you have a proper example? How would a use know the JNLP URL?

Reply:
The user wouldn't know this and ideally shouldn't configure the JNLP URL, its too much to push on the user and I haven't seen a case where a user requested a custom url.

Right now its derived from here: https://github.com/jenkinsci/mesos-plugin/blob/1.x/src/main/java/org/jenkinsci/plugins/mesos/JenkinsScheduler.java#L216-L222

I'm in favour of not exposign this URL unless there's concrete use case from the user to change it.